plugins/github-actions/src/components/Widget/Widget.tsx

```diff
@@ -108,3 +108,54 @@ export const Widget = ({
     </InfoCard>
   );
 };
+
+const WidgetListContent = ({
+  error,
+  loading,
+  branch,
+}: {
+  error?: Error;
+  loading?: boolean;
+  lastRun: WorkflowRun;
+  branch: string;
+}) => {
+  if (error) return <Typography>Couldn't fetch {branch} runs</Typography>;
+  if (loading) return <LinearProgress />;
+  return <WorkflowRunsTable />;
+};
+
+export const WidgetList = ({
```

Comment:
maybe name it more descriptively along the lines of `RecentWorkflowRunsCard` - that represents what it is and what it does better, I think

Reply:
Maybe `WidgetRecentWorkflowRunsCard`. What do you think of this name @freben ?

Reply:
If you absolutely want the word Widget in there, sure :) I was thinking that a card is a card, so to speak.

Reply:
Ok, I think having the work widget in it make the purpose more clear.

Reply:
The name is to long with Widget on it. So I am dropping the widget :P